Libraries/Image/RCTNetworkImageView.m

```diff
@@ -62,10 +62,7 @@ - (void)setImageURL:(NSURL *)imageURL resetToDefaultImageWhileLoading:(BOOL)rese
       _downloadToken = [_imageDownloader downloadDataForURL:imageURL block:^(NSData *data, NSError *error) {
         if (data) {
           CAKeyframeAnimation *animation = RCTGIFImageWithData(data);
-          CGImageRef firstFrame = (__bridge CGImageRef)animation.values.firstObject;
-          self.layer.bounds = CGRectMake(0, 0, CGImageGetWidth(firstFrame), CGImageGetHeight(firstFrame));
-          self.layer.contentsScale = 1.0;
-          self.layer.contentsGravity = kCAGravityResizeAspect;
+          self.layer.contentsScale = RCTScreenScale();
```

Comment:
@a2 - good point, let's just make this 1.0 and we can open a separate issue for `@2x`+ support if you like